### PR TITLE
Update SSO login.

### PR DESCRIPTION
**Pull request by YKo20010**
Logging in through SSO now works. Next step is to add logout.

### CODE REVIEW

File: src/components/PollingApp/index.tsx [hunk 1]
```diff
@@ -92,6 +92,28 @@ class PollingApp extends React.Component<any, PollingAppState> {
...
+                rememberCurrentUser(currentUser);
+                this.props.dispatch({ type: 'set-user', user: currentUser });
+                this.setState({ isLoading: false });
+            } catch (error) {
+                console.log(error);
+                forgetCurrentUser();
+                this.setState({ isLoading: false, showLoginError: true });
+            }
+        }
+        console.log(getCurrentUserLocal());
```
Comment (danielVebman) [inline src/components/PollingApp/index.tsx:114]: Should this print and the one on line 99 be in master?

File: src/components/PollingApp/index.tsx [hunk 2]
```diff
@@ -21,7 +20,7 @@ import {
     currentUserExists,
     forgetCurrentUser,
     getCurrentPoll,
-    getCurrentUser,
+    getCurrentUser as getCurrentUserLocal,
```
Comment (danielVebman) [inline src/components/PollingApp/index.tsx:23]: Maybe name this `getCurrentLocalUser`? At least that's my understanding of what this method does. If I'm wrong, I think another name should clarify.

File: src/components/PollingApp/index.tsx [hunk 3]
```diff
@@ -92,6 +92,28 @@ class PollingApp extends React.Component<any, PollingAppState> {
...
 
+    public getCurrentUser = async () => {
+        if (!currentUserExists()) {
+            try {
+                const user = await getCurrentUserRequest();
+                console.log(user);
+                const currentUser = {
+                    id: user["id"],
+                    name: user["name"],
+                    netId: user["netID"],
```
Comment (danielVebman) [inline src/components/PollingApp/index.tsx:103]: `user.id` `user.name` `user.netID`?

File: src/components/PollingApp/index.tsx [hunk 4]
```diff
@@ -187,7 +209,7 @@ class PollingApp extends React.Component<any, PollingAppState> {
                 await generateUserSession(response.tokenId);
                 const adminSessions = await condenseAdminSessions();
                 this.props.dispatch({ type: 'set-sessions', sessions: adminSessions });
-                const currentUser = getCurrentUser();
+                const currentUser = getCurrentUserLocal();
```
Comment (danielVebman) [inline src/components/PollingApp/index.tsx:212]: semicolons 😓

**Review by danielVebman — APPROVED** [review]
Just a couple really small things

**Review by ywsang — APPROVED** [review]
lgtm once daniel's comments are addressed!

File: src/components/PollingApp/index.tsx (hunk 4 above)
Comment (YKo20010) [reply src/components/PollingApp/index.tsx:212]: Everything else came with semicolons so Im just tryna stay consistent 👀

**Pull request merged**